server/routes/activities/waitlist-application/handlers/activity.test.ts

```
@@ -115,6 +121,32 @@ describe('Route Handlers - Waitlist application - Request date', () => {
         'Alan Key is already allocated or on the waitlist for test activity',
       )
     })
+
+    it('should throw validation error if prisoner already on a waitlist', async () => {
```

Perhaps have coverage for APPROVED as well, as in should throw an error?